### ISSUE TITLE
Possible Forge Port

### ISSUE THREAD

**Issue by ghost** [BluCobalt/realmsfix#3]
Do you think the mod could be ported to forge due to it's simple mixin? Not entirely a request, but I'm curious.

**Comment by BluCobalt** [comment]
Definitely, but I don't have any experience with forge. But, as you said, it IS only a simple mixin so it shouldn't be too hard for me to figure out.
